miqcli/constants.py

```diff
@@ -27,6 +27,7 @@
 DEFAULT_CONFIG = {
     'username': 'admin',
     'password': 'smartvm',
-    'url': 'https://localhost:8443',
+    'url': 'https://localhost:8443/api',
```

Comment:
Users now will have to provide the keyword **/api** to their MIQ URL if they wish to override the default settings? What if we handle adding the api keyword when creating the endpoint url to generate the authentication token? This way then the user does not need to provide it. I am assuming if the user does not provide the api keyword in their URL the request call would fail above since the endpoint is not correctly set.

We can always check before adding the api keyword to the endpoint to see if the user already included it. That way we do not append it again.

What do you think?